content/english/resources/bsl3.md

```diff
@@ -1,33 +1,47 @@
 ---
 title: "BSL3 Biomedicum-SciLifeLab Collaborative Platform"
 category: "plp1"
-banner: "/resources_thumbs/bsl3_lab.jpg"
```

Comment:
I see the `bsl3_lab.jpg` is not removed, if it is not used elsewhere can we please remove it?

Reply:
We can delete